lib/Pipeline.js

```diff
@@ -82,14 +83,22 @@ Pipeline.processJSON = function(gltf, options) {
  */
 Pipeline.processJSONWithExtras  = function(gltfWithExtras, options) {
     options = defaultValue(options, {});
+
+    addDefaults(gltfWithExtras, options);
+
+    // Generate stats for unoptimized input
+    if (options.stats) {
+        gltfStatistics.printStatistics(gltfWithExtras, options.inputPath);
+    }
```

Comment:
`gltfStatistics` would need some tweaks in order to work if some sections of the glTF are missing. Like if `gltf.images` is undefined `getNumberOfProperties` will throw an error.